libs/etcdlib/test/etcdlib_test.c

```diff
@@ -98,6 +97,7 @@ int waitforchangetest() {
 	sleep(1);
 	etcdlib_set(etcdlib, "hier/ar/chi/cal", "testvalue3", 5, false);
 	void *resVal = NULL;
+	printf("joining\n");
```

Comment:
Can be removed I think?

Reply:
It helped me in detecting an issue where etcdlib_watch was blocking other threads, made me see that one thread was waiting on the others.

Reply:
I overlooked that this is a test, in that case I'm fine with it.